docs/language/crypto.md

```diff
@@ -99,13 +106,12 @@ The API of the Crypto contract is:
 ```cadence
 pub contract Crypto {
 
-    pub struct SignatureAlgorithm {
-        pub let name: String
-    }
-
     // Hash the data using the given hashing algorithm and returns the hashed data.
     pub fun hash(_ data: [UInt8], algorithm: HashAlgorithm): [UInt8]
 
+    // Hash the data using the given hashing algorithm and the tag. Returns the hashed data.
+    pub fun hashWithTag(_ data: [UInt8], tag: string, algorithm: HashAlgorithm): [UInt8]
```

Comment:
I have suggested `hashWithTag` as I wasn't sure if `PublicKey.Verify` would be implemented entirely by the fvm or would be partly implemented in Cadence code. In the case of the former, I don't think `hashWithTag` is needed in Cadence, we would only need it as an fvm function.